karate-tests/src/test/java/mocks/sandbox/stubs.js

```diff
@@ -8,7 +8,6 @@
 const SEARCH_PATIENT_9000000009 = context.read('classpath:mocks/stubs/searchResponses/search_patient_9000000009.json')
 const SEARCH_PATIENT_9000000017 = context.read('classpath:mocks/stubs/searchResponses/search_patient_9000000017.json')
 const RESTRICTED_PATIENT_SEARCH = context.read('classpath:mocks/stubs/searchResponses/search_patient_9000000025.json')
-const COMPOUND_NAME_SEARCH = context.read('classpath:mocks/stubs/searchResponses/compound_name_search.json')
```

Comment:
just to confirm, is this now being tested by the JOHN_PAUL_SMITH_SEARCHSET in get-patient-search.js? As long as we're still testing the handling of compound names, that's the main thing.

Reply:
Yes - it's the same file contents, I just gave the constant a different name